src/Authenticator/LoginHandler.php

```diff
@@ -147,16 +158,21 @@ public function startRegistration(HTTPRequest $request)
             );
         }
 
+        $method = $this->getMethodRegistry()->getMethodByURLSegment($request->param('Method'));
+
         // If the user isn't fully logged in and they already have a registered method, they can't register another
-        if (is_null($loggedInMember) && $sessionMember && count($sessionMember->RegisteredMFAMethods()) > 0) {
+        // provided that they're not registering a backup method
+        $registeredMethodCount = $sessionMember->RegisteredMFAMethods()->count();
+        $isRegisteringBackupMethod =
+            get_class($method) === Config::inst()->get(MethodRegistry::class, 'default_backup_method');
```

Comment:
This won't be extensible - perhaps you could do `is_a($method, MethodRegistry::config()->get('default_backup_method'), true)` which would allow subclasses too

Reply:
Sure. It'd be pretty weird to have parent/sub-classes on your MFA methods but 🤷‍♂️